packages/google_maps_flutter/google_maps_flutter_web/lib/google_maps_flutter_web.dart

```diff
@@ -31,6 +32,7 @@ part 'src/convert.dart';
 part 'src/google_maps_controller.dart';
 part 'src/google_maps_flutter_web.dart';
 part 'src/marker.dart';
+part 'src/marker_clustering.dart';
```

Comment:
Does this really really need to be a `part`? Can it be `import`ed?

Reply:
Most likely it can be imported as well with minor changes, using part was chosen to follow the design of this package as everything else is implemented as partof syntax.

Reply:
marker_clustering.dart is now imported.
it uses shared conversion methods from the convert.dart file.
these methods were private, and I needed to change them as public to be able to use them from the maker_clustering.dart file.